### PR TITLE
Data in Home Fragment

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
Create Home Fragment UI ,Tash Obj and repository base .
Redmine : https://edu-redmine.framgia.vn/issues/5309

### CODE REVIEW

**Comment by phungvankien1997** [comment]
ready

**Comment by phungvankien1997** [comment]
ready

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/java/com/vn/kienphung/music_52/data/model/Track.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Track implements Parcelable {
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/vn/kienphung/music_52/data/model/Track.java]: remove bl

**Comment by meoconlontonbebong** [comment]
ready

**Pull request merged**